eq-author/src/App/page/Routing/RoutingPage/RoutingEditor/RuleEditor/BinaryExpressionEditor/checkboxRouting.test.js

```diff
@@ -151,4 +151,13 @@ describe("Checkbox routing", () => {
       "AnyOf"
     );
   });
+
+  it("does not render 'choose' button or selected options when match condition is unanswered", () => {
+    props.expression.condition = "Unanswered";
+    const { queryByText } = renderRouting();
+
+    expect(queryByText("CHOOSE")).toBeFalsy();
```

Comment:
Could we do a check first to see that `CHOOSE` is there before setting `condition` to `Unanswered`? I think it makes the test more trustworthy. So if the test is not working, that approach should pick it up.